source/components/index.rst

```diff
@@ -57,6 +57,18 @@ The following TextDecorations are available:
 * Underlined
 * Obfuscated
 
+Additionally, you can set a font contained within the client's texture pack or within
+their Minecraft jar at ``assets\minecraft\font``.
```

Comment:
This feels like it should really be part of the javadoc, rather than part of the prose documentation -- it doesn't do much beyond listing the method

Reply:
Then shouldn't the section be renamed from "Styling components" to, say, "Commonly used styling" as not to let the reader believe they are limited to color and decoration?

Or did you mean listing the filepath is an issue? I could list the actual file names but it varies between supported Minecraft versions.